src/managed_data/api.py

```diff
@@ -9,15 +9,125 @@
 
 from beartype import beartype
 from deeporigin.exceptions import DeepOriginException
+
+# this import is to allow us to use functions
+# not marked in __all__ in _api
 from deeporigin.managed_data import _api
-from deeporigin.managed_data.client import Client
-from deeporigin.managed_data.schema import DatabaseReturnType, IDFormat
-from deeporigin.utils import PREFIXES
 
+# this import is to make sure that all simply-wrapped
+# functions in _api are available in api (this module)
+from deeporigin.managed_data._api import *  # noqa: F403
```

Comment:
If I understand correctly, this re-exports functions from `_api` to this file? If so, I understand the sentiment, but, personally, I would prefer to expose these separately. I find it useful where there is a 1-1 mapping between an HTTP API and a client library. When use API extensions get merged with the 1-1 mapping, it makes it harder for me to understand.

Reply:
>  I would prefer to expose these separately. 

you can use them separately, using `...import _api`.

Reply:
That means there'd be 2 ways to do the same thing, right?